crates/rolldown/src/types/module_factory.rs

```diff
@@ -93,8 +93,8 @@ impl<'a> CreateModuleContext<'a> {
   pub async fn resolve_dependencies(
     &mut self,
     dependencies: &IndexVec<ImportRecordIdx, RawImportRecord>,
-  ) -> anyhow::Result<IndexVec<ImportRecordIdx, ResolvedId>> {
-    let jobs = dependencies.iter_enumerated().map(|(idx, item)| {
+  ) -> anyhow::Result<Result<IndexVec<ImportRecordIdx, ResolvedId>, Vec<BuildDiagnostic>>> {
```

Comment:
https://github.com/rolldown/rolldown/blob/d023f2f95d604a3ab2ce0d627795fc8a3b77bd99/crates/rolldown_error/src/type_aliases.rs#L3-L3, use this, please.

Reply:
Yes, I remember this, but I forgot where it was. 😂